app/src/main/java/com/iitdh/sonusourav/instigo/Utils/CommonFunctions.java

```diff
@@ -144,15 +150,33 @@ public static void setUser(Activity activity){
 
             TextView emailTextView = drawerHeader.findViewById(R.id.nav_header_email);
             TextView usernameTextView = drawerHeader.findViewById(R.id.nav_header_username);
+            ImageView letterImage = drawerHeader.findViewById(R.id.nav_header_letterView);
             ImageView userImage=drawerHeader.findViewById(R.id.nav_header_imageView);
 
             emailTextView.setText(firebaseUser.getEmail());
             usernameTextView.setText(firebaseUser.getDisplayName());
-            Glide.with(activity.getApplicationContext())
-                    .load(firebaseUser.getPhotoUrl())
-                    .into(userImage);
-            Log.d("Nav_Image","Reached");
 
+            Uri photoUri = firebaseUser.getPhotoUrl();
+            if(photoUri == null){
+                String userName = firebaseUser.getDisplayName();
+                char ch;
+                if(userName == null)
+                    ch = 'A';
```

Comment:
Can we add a dummy profile icon instead of 'A' character? Display it in circleImageView and make textView invisible.